api/proto/auth/config.go

```diff
@@ -0,0 +1,98 @@
+package auth
+
+import (
+	"context"
+	"crypto/tls"
+	"net"
+	"time"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/lib/utils"
+	"github.com/gravitational/trace"
+)
+
+// ClientConfig contains configuration of the client
+type ClientConfig struct {
+	// Addrs is a list of addresses to dial
+	Addrs []utils.NetAddr
```

Comment:
Let's see if we can't get rid of the `utils.NetAddr` and then remove the dependency on the `utils` package.